packages/vanilla/src/components/card/generalCard.stories.js

```diff
@@ -16,51 +16,52 @@ export default {
       name: 'Modifier',
       description: 'Set the modifier class of the card',
       control: 'select',
-      options: ['default', 'info', 'success', 'danger']
+      options: ['default', 'info', 'success', 'danger', 'warning']
+    },
+    showIcon: {
+      name: 'Show Icon',
+      description: 'Show an icon in the title',
+      control: 'boolean'
+    },
+    bodyText: {
+      name: 'Body Text',
+      description: 'Set the body text of the card',
+      control: 'text'
     }
   }
 };
 
-const DefaultGeneralCardTemplate = ({ title, modifier }) => {
+const DefaultGeneralCardTemplate = ({ title, modifier, showIcon, bodyText }) => {
   return `
     <div class="wrapper" style="display: grid; grid-template-columns: 432px">
       <div class="cbp-card ${modifier != 'default' ? `cbp-card--${modifier}` : ''}">
         <div class="cbp-card__content">
           <div class="cbp-card__header">
-            <h4 class="cbp-card__title"><i class="fas fa-info-circle cbp-margin-right-2x"></i>${title}</h4>
+            <h4 class="cbp-card__title">
+              ${showIcon ? '<i class="fas fa-info-circle cbp-margin-right-2x"></i>' : ''}
```

Comment:
Sorry this is nitpicky, but make sure to put the icon and the heading/title on the same line next to each other. The line break is showing up as a space and adding to the margin-right making a larger gap than intended. I've fixed a bunch of these when editing stories myself.